velox/exec/tests/SpillerTest.cpp

```diff
@@ -194,16 +194,12 @@ class SpillerTest : public exec::test::RowContainerTestBase,
           }
           const int partitionNum =
               hashBits_.partition(hashes[0], numPartitions_);
-          if (numRowsPerPartition[partitionNum] == 0) {
-            selectedRows.setValid(index, false);
-            continue;
+          for (int i = 0;
+               i < numDuplicates && --numRowsPerPartition[partitionNum] >= 0;
+               ++i) {
+            rowVector_->copy(batch.get(), numFilledRows++, index, 1);
```

Comment:
nit: Perhaps, add a comment to explain which rows are being copied to rowVector_. I'm not sure I understand this code.